Server/database/query/Bills/getLastBill.js

```diff
@@ -0,0 +1,10 @@
+const connection =require('../../connection')
+
+module.exports =  (type, limit ) => {
+    const sql = {
+        text: 'SELECT gid, users_id, provider_id , type , total_amount , bill_DATE  , due_DATE , start_DATE  , end_DATE , bill_Number  FROM bill   WHERE type=$1  ORDER BY bill_DATE desc  limit=$2'
```

Comment:
as we discussed before, we want to return all the bills for that type without limit with numbers, then we can handle it in the front end, and in this way, we don't need another request for the date, just make a filter depends on the date property.